parser/MzIdParser.py

```diff
@@ -765,6 +763,12 @@ def write_new_upload(self):
             'identification_file_name': os.path.basename(self.mzid_path),
         }
         self.writer.write_data('Upload', upload_data)
+        # table = SATable('upload', self.writer.meta, autoload_with=self.writer.engine, quote=False)
+        # with self.writer.engine.connect() as conn:
+        #     statement = table.insert().values(upload_data).returning(table.columns[0])  #  RETURNING id AS upload_id
+        #     result = conn.execute(statement)
+        #     self.writer.upload_id = result.fetchall()[0][0]
+        #     conn.close()
```

Comment:
This is leftover from trying to convert to sqlalchemy?

Reply:
no, its left over from trying to update the parser on the xiView public site.
There, we have a different system for unique upload id's and a two step process for writing things to the upload table.
That's why i needed this different code for writing data which has this "RETURNING" bit in it (it returns the new auto-incremented id).
Updating the parser on the public xiView prob isn't a priority at moment so this can prob be forgotten about for time being